pkg/kubectl/cmd/describe.go

```diff
@@ -36,8 +36,7 @@ import (
 // DescribeOptions is the start of the data required to perform the operation.  As new fields are added, add them here instead of
 // referencing the cmd.Flags()
 type DescribeOptions struct {
```

Comment:
This one too (you get the idea, I'll stop commenting on these, but please do them all...)